### DISCUSSION TITLE
listener - smtp server

### DISCUSSION THREAD

**Discussion by onhru** [canarytrace/documentation#195] [Listener]
Dobry den,

mam v listeneru nastaven SMTP server, tedy adresu a port znam, ale uzivatel a heslo neni vyzadovano ve vnitrni siti. tedy tyto hodnoty nejsou zadany. 
V listener logu mam tuto informaci:
Report to e-mail will not be send, because some e-mail configuration parameter is missing.

vzhledem k tomu ze uzivatel a heslo neni nutne zadavat, jak to mam udelat aby se tyto data v ramci listeneru nevyzadovala.

diky 
Ondrej 
(Direct-auto)

**Comment by rdpanek** [comment]
Dobrý den, to je dobrý point. Upravíme konfiguraci na pouhé varování.

**Comment by rdpanek** [comment]
@onhru upraveno ve verzi `quay.io/canarytrace/listener-agent:1.46` je to nyni OK?

**Reply by onhru** [reply]
zmenila se hlaska na:
⚠️  The report might not be sent via email because some email configuration parameters are missing.

ale ve vysledku se nic nezmenilo

**Reply by rdpanek** [reply]
a co ma byt vysledkem? nechodi emaily nebo co melo byt jinak?

**Reply by onhru** [reply]
predpokladam ze zacnou chodit emaily, kdy nebude vyzadovano toto nastaveni:
            - name: EMAIL_SMTP_USER
              value: ""
            - name: EMAIL_SMTP_PASS
              value: ""

**Reply by onhru** [reply]
Dobry den, chtel bych se zeptat jak to vypada?

**Reply by rdpanek** [reply]
A jake emaily vam nechodi, nejake chybove hlasky se zobrazuji?

**Reply by onhru** [reply]
no nechodi zadne, protoze nejsou vyplneny vsechny udaje. potrebuji aby se netrvalo na loginu a heslu. pokud SMTP server nevyzaduje tyto udaje. nemelo by byt nastaveni povinne.
viz vyse

**Reply by rdpanek** [reply]
Ale to jsem Vam jiz psal, ze v 1.46 se zobrazuje pouze varovani https://github.com/canarytrace/documentation/issues/199 pak v 1.47 jsme pridali jeste moznost vypnout vyzadovanou vlastnost secure https://github.com/canarytrace/documentation/issues/201 zkuste posledni 1.47 tam uz muzete ovlivnit kompletni nastaveni.

- https://v2doc.canarytrace.com/docs/listener/releases
- https://v2doc.canarytrace.com/docs/listener/agent#reporters-1

**Reply by onhru** [reply]
dekuju, vypada ze to funguje .
jinak v dokumentaci na strance relases , verze 1.47 to vede na stranku kde je 404 , 
 https://v2doc.canarytrace.com/docs/listener/listener/agent#reporters-1
 nicemene az link pod cislem #201 je funkcni
